packages/components/src/components/hds/copy/button/index.ts

```diff
@@ -7,17 +7,32 @@ import Component from '@glimmer/component';
 import { assert } from '@ember/debug';
 import { tracked } from '@glimmer/tracking';
 import { action } from '@ember/object';
+import { HdsCopyButtonSizeValues } from './types.ts';
+import type { HdsCopyButtonSizes } from './types.ts';
+import type { HdsButtonSignature } from '../../button/';
+import type { ModifierSignature } from '../../../../modifiers/hds-clipboard.ts';
```

Comment:
If you rename the signature to `HdsClipboardModifierSignature` in the other PR, remember to update also this import.